cephfs/cephfs_test.go

```diff
@@ -15,14 +16,60 @@ import (
 )
 
 var (
-	CephMountTest = "/tmp/ceph/mds/mnt/"
+	CephMountDir     = "/tmp/ceph/mds/mnt/"
+	requireCephMount = false
+	testMdsName      = "Z"
 )
 
+func init() {
+	mdir := os.Getenv("GO_CEPH_TEST_MOUNT_DIR")
+	if mdir != "" {
+		CephMountDir = mdir
+	}
+	reqMount := os.Getenv("GO_CEPH_TEST_REQUIRE_MOUNT")
+	if reqMount == "yes" || reqMount == "true" {
```

Comment:
I'd rather have the user explicitly ask for it to be activated. Maybe I'm being too much of a mind reader but I am assuming that outside of our CI most people running the tests would prefer the skip.

Reply:
Sure. What I proposed still needs the user to set something to activate it. But if the user sets something but it is not "no" or "false" it would be interpreted as yes.